src/hotspot/share/prims/jvmti.xml

```diff
@@ -742,7 +742,10 @@ Agent_OnUnload_L(JavaVM *vm)</example>
         <li>The pair of quote marks is not included in the option.</li>
     </ul>
     <code>JNI_CreateJavaVM</code> (in the JNI Invocation API) will prepend these options to the options supplied
-    in its <code>JavaVMInitArgs</code> argument. Platforms may disable this feature in cases where security is
+    in its <code>JavaVMInitArgs</code> argument. Note that module related options must be expressed in their
+    "option=value" form (not "option value") for <code>JNI_CreateJavaVM</code> to process them correctly.
```

Comment:
This looks okay. I'm just comparing it to the text that we put into the JNI spec:

"The module related options ... as option strings using their "option=value" format instead of their "option value" format. (Note the required = between "option" and "value".)"

It uses "format" instead of "form" and also, the bit I think words well, is to point the out the "required =" to force the reader to re-read the previous sentence and see what the difference is in the formats.

Reply:
I agree with not duplicating the entire sentence but what would you think about replacing the text in the parenthesis  with "Note the required = between option and value". Only asking because "(not "option value")" isn't as clear that we want the reader to see that you can't use a space here.

Reply:
Okay. I thought it very clear to say "use format X (not format Y)", but sure I can change it.